<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>190</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="chooseDataFileLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose data file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadDataFileButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load TTL File</string>
   </property>
  </widget>
  <widget class="QLabel" name="chosenDataFile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choosen Validation Rule Set:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chosenValidatorFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Geometry Literal Content Validation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GeoSPARQL 1.0 Validation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GeoSPARQL 1.1 Validation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GeoSPARQL Extended Rule Set Validation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TTL Validator</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="startValidationButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Validation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
